components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -347,9 +348,15 @@ class GleanTest {
         Glean.initialized = true
     }
 
-    @Test(expected = IllegalStateException::class)
-    fun `Don't initialize twice`() {
+    @Test
+    fun `Initializing twice is a no-op`() {
+        val beforeConfig = Glean.configuration
+
         Glean.initialize(ApplicationProvider.getApplicationContext())
```

Comment:
nit: Should we explicitly provide a configuration with test values here (e.g. the server url to something like "https:\\no.init.should.take.place.with.this"?)?